analysis/descriptives/Stage2_Missing_Table1.R

```diff
@@ -1,22 +1,15 @@
 ## =============================================================================
-## Project:     Post covid unvaccinated project
-##
 ##
 ## Purpose: 
 ##  Apply stage 2. Create missing data, range and descriptive statistics tables
 ##  - Table 0 - Missing data and range
 ##  - Table 1 - Descriptive statistics
 ## 
-## Authors: Lucy Teece (adapted from original written by Genevieve & Rochelle for vaccinated project)
-## Reviewer: Yinghui Wei
-## 
 ## Content: 
 ## 0. Load relevant libraries and read data/arguments
-## 1. Output missing data and range check tables
+## 1. Output missing data  and range check tables
 ## 2. Output table 1
 ## 
-## NOTE: This code outputs 3 .csv files and 1 R dataset
-##  
```

Comment:
Although I like to find comments in the same script file, I think there was an agreement to remove comments from the beginning of scripts and create an md file instead.
This applies on all files containing comments at the beginning.

Reply:
I'll add this as an issue and update at a later date as tis doesn't affect the running of the repo